From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rishwanth Yeddula <rish@cpanel.net>
Date: Sat, 15 Apr 2017 09:45:34 -0500
Subject: [PATCH 1/5] Patch build files to use SCL libuv paths

Adopted from the passenger-4.0.38-libeio patch from the rh-passenger40
SCL package.

The 5.x series of passenger introduced new symbols to the bundled libev,
and is therefore incompatible with the upstream libev packages -
therefore this patch only updates the libuv paths.
---
 build/common_library.rb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/common_library.rb b/build/common_library.rb
index 9991caeec..5d3036f3b 100644
--- a/build/common_library.rb
+++ b/build/common_library.rb
@@ -223,12 +223,12 @@ def libuv_libs
     "#{LIBUV_OUTPUT_DIR}.libs/libuv.a #{$1}".strip
   end
 else
-  LIBUV_CFLAGS = string_option('LIBUV_CFLAGS', '-I/usr/include/libuv')
+  LIBUV_CFLAGS = string_option('LIBUV_CFLAGS', '-I/opt/cpanel/ea-ruby24/root/usr/include/')
   LIBUV_TARGET = nil
   task :libuv  # do nothing
 
   def libuv_libs
-    string_option('LIBUV_LIBS', '-luv')
+    string_option('LIBUV_LIBS', '-L/opt/cpanel/ea-ruby24/root/usr/lib64 -luv')
   end
 end
 
